ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1467,6 +1467,17 @@ objects:
           done
 
           echo "bootstrap: ${BOOTSTRAP_IP} control-plane: ${CONTROL_PLANE_0_IP} ${CONTROL_PLANE_1_IP} ${CONTROL_PLANE_2_IP} compute: ${COMPUTE_0_IP} ${COMPUTE_1_IP}"
+        elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+          # run setup job
+          echo "setting up openstack upi job"
+
+          # TODO: get openstack upi dependancies
+          # TODO: create dns records
+          # TODO: create install config
+          # TODO: create and manage manifests
+          # TODO: generate and manage ignition configs
+          # TODO: deploy openstack upi cluster 
+
```

Comment:
It seems that one of the reasons, in ipi, for the separate template was the reigstration of DNS would break aws. Move the dns registration to under one of the CLUSTER_TYPE == openstack code blocks, so hopefully we can use the same template now. 
see     https://github.com/openshift/release/commit/bbd38849